resources/lib/util/moonlighthelper.py

```diff
@@ -59,20 +59,21 @@ def list_games(self):
         list_regex = r'\d+\.\s+([^\n]*)'
 
         try:
-            moonlightOut = subprocess.check_output(['moonlight', 'list'], cwd=binary_path, timeout=5, encoding='utf-8', start_new_session=True)
+            host = self.plugin.getSetting('host_addr')
+            moonlightOut = subprocess.check_output(['moonlight', 'list', host], cwd=binary_path, timeout=5, encoding='utf-8', start_new_session=True)
 
```

Comment:
This adds the configured host address as argument to moonlight. Without it, Luna would err.